grpc-interface-support/test/integration/test_integration_cpp.py

```diff
@@ -80,6 +80,10 @@ def test__integration():
     ensure_package_is_generated()
     assert get_dependency_count("seats-service-sdk") == 1
     assert get_dependency_count("hornservice-service-sdk") == 1
+    assert get_dependency_count("val-service-sdk") == 1
+    assert get_dependency_count("bcmdoorservice-service-sdk") == 1
+    assert get_dependency_count("vcsptcpbylimservice-service-sdk") == 1
+    assert get_dependency_count("vcsmotortrqmngservice-service-sdk") == 1
```

Comment:
what is happening with the names here? :D
no issue, just find it funny

Reply:
its the name of the service :)